nightingale_payload/src/main.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+
+import rospy
+import tf2_ros
+import tf_conversions
+import numpy as np
+
+from nightingale_msgs.msg import Payload
+from sensor_msgs.msg import JointState
+from urdf_parser_py.urdf import Robot
+
+
+class PayloadEstimator:
+    GRAVITY = 9.8
+
+    def __init__(self):
+        rospy.init_node("payload_estimator_node")
+
+        self.robot = Robot.from_parameter_server()
+        rospy.loginfo(f"{self.robot.link_map['right_shoulder_link'].inertial}")
+
+        self.forces = np.zeros(6)
+        self.last_forces = np.zeros(6)
+        self.filter_coeff = 0.5
+
+        self.tf_buffer = tf2_ros.Buffer()
+        self.tf_listener = tf2_ros.TransformListener(self.tf_buffer)
+
+        self.payload_pub = rospy.Publisher(
```

Comment:
What exactly does this package do? I thought it senses the weight of the box and its contents. If we are basically querying the weight would a service do better here?